### PR TITLE
[ar] Localize term "Pod" to Arabic

### PR DESCRIPTION
**Pull request by adowair**
### Describe your changes

This PR will add a new file `content/ar/pod.md` localizing the glossary entry
"pod" into Arabic.

### Related issue number or link (ex: `resolves #issue-number`)

No issue assigned.

### Checklist before opening this PR (put `x` in the checkboxes)
- [x] This PR does not contain plagiarism
  - don’t copy other people’s work unless you are quoting and contributing it to them.
- [x] I have signed off on all commits 
  - [signing off](https://docs.github.com/en/authentication/managing-commit-signature-verification/signing-commits) (ex: `git commit -s`) is to affirm that commits comply [DCO](https://wiki.linuxfoundation.org/dco). If you are working locally, you could add an alias to your `gitconfig` by running `git config --global alias.ci "commit -s"`.

### CODE REVIEW

**Comment by adowair** [comment]
Preview after commit b7c390f

**Comment by adowair** [comment]
Screenshot of page render after commit 4ce624b

**Comment by adowair** [comment]
@TarekMSayed @arezk84 d you have any additional comments or suggestions? I appreciate your effort

File: content/ar/pod.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+حتى يعمل التطبيق كما ينبغي.
+تُسَهِّل الحجيرات إدارة تلك التطبيقات
+حيث أنها تُحَزِّم عدة حاويات في كيانٍ واحد سهل الإدارة.
+
+## كيف تساعد؟
+من أمثلة الحُجَيرات متعددة الحاويات:
+* حُجَيرة تحتوي على حاوية تقوم ببعض الإعدادات الأولية في قرينتها
+* حُجَيرة فيها حاوية مساعدة (sidecar) تعمل بالتزامن مع الحاوية الأساسية للقيام بوظيفة مساندة:
+	* جمع السجلات ([logs](https://en.wikipedia.org/wiki/Logging_(computing))) من الحاوية الأساسية
+	* إدارة تواصل الحاوية الأساسية مع النظام (انظر إلى [الشبكة الخدمية service mesh](/service-mesh/))
```
Comment (TarekMSayed) [inline content/ar/pod.md:28]: طالما هناك رابط للمصطلح فلا داعي لذكر الإنجليزية

File: content/ar/pod.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+title: حُجَيرة
+status: مكتملة
+category: مفهوم
+---
+
+الحُجَيرة (Pod) في سياق [الكوبرنيتيز](/kubernetes/)
+هي الوحدة الإدارية الأساسية لتشغيل التطبيقات.
+تتكون الحُجَيرة من [حاوية](/ar/container/) أو عدة حاويات بالإضافة إلى ملحقاتها.
+يدير [عنقود](/ar/cluster/) كوبرنيتيز الحجيرات في مجموعات مستقلة تخدم كلاً منها تطبيق معين.
+تلك المجموعات قابلة [للتوسع](/ar/auto-scaling/) [الأفقي](/horizontal-scaling/) [والعامودي](/vertical-scaling/) حسب احتياج التطبيق للموارد الحاسوبية.
```
[the remark this answers is not in the document]
Comment (TarekMSayed) [reply content/ar/pod.md:11]: نعم عمودي بدون المد والأفضل أن تستخدم رأسي

File: content/ar/pod.md (hunk 1 above)
Comment (adowair) [reply content/ar/pod.md]: تمام شكراً على التعليق. حذفت في: https://github.com/cncf/glossary/pull/2807/commits/1a23d2116b70a363ab3b0ed20407639ce0250f03.

File: content/ar/pod.md [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+فعلى مدير النظام الحفاظ على صحة جميع العناصر،
+وضمان قدرة التواصل بينها، 
+حتى يعمل التطبيق كما ينبغي.
+تُسَهِّل الحجيرات إدارة تلك التطبيقات
+حيث أنها تُحَزِّم عدة حاويات في كيانٍ واحد سهل الإدارة.
+
+## كيف تساعد؟
+من أمثلة الحُجَيرات متعددة الحاويات:
+* حُجَيرة تحتوي على حاوية تقوم ببعض الإعدادات الأولية في قرينتها
+* حُجَيرة فيها حاوية مساعدة (sidecar) تعمل بالتزامن مع الحاوية الأساسية للقيام بوظيفة مساندة:
```
[the remark this answers is not in the document]
Comment (adowair) [reply content/ar/pod.md]: شكراً على التعليق. بالإشارة الى حديثي مع @TarekMSayed [هنا](https://cloud-native.slack.com/archives/C02UFT8V256/p1706815211482249?thread_ts=1705827628.657469&cid=C02UFT8V256), اقتُرِحَ استخدام كلمة "جانبية" من قبل، ونظراً لاقتراحها الآن أكثر من مرة، قمت باسخدامها: https://github.com/cncf/glossary/pull/2807/commits/1a23d2116b70a363ab3b0ed20407639ce0250f03.

File: content/ar/pod.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (adowair) [reply content/ar/pod.md]: شكراً على التعليق، تم التغيير إلي "عمودي" هنا: https://github.com/cncf/glossary/pull/2807/commits/1a23d2116b70a363ab3b0ed20407639ce0250f03.

File: content/ar/pod.md [hunk 4]
```diff
@@ -0,0 +1,31 @@
+---
+title: حُجَيرة
+status: مكتملة
+category: مفهوم
+---
+
+الحُجَيرة (Pod) في سياق [الكوبرنيتيز](/kubernetes/)
```
[the remark this answers is not in the document]
Comment (adowair) [reply content/ar/pod.md]: Yes, thank you @seifrajhi—done here: https://github.com/cncf/glossary/pull/2807/commits/99d9d99f26285fb78bdd847e003a00dc7e288186.

File: content/ar/pod.md [hunk 5]
```diff
@@ -0,0 +1,31 @@
...
+
+الحُجَيرة (Pod) في سياق [الكوبيرنيتيس](/kubernetes/)
+هي الوحدة الإدارية الأساسية لتشغيل التطبيقات.
+تتكون الحُجَيرة من [حاوية](/ar/container/) أو عدة حاويات بالإضافة إلى ملحقاتها.
+يدير [عنقود](/ar/cluster/) كوبرنيتيز الحجيرات في مجموعات مستقلة تخدم كلاً منها تطبيق معين.
+تلك المجموعات قابلة [للتوسع](/ar/auto-scaling/) [الأفقي](/horizontal-scaling/) [والعمودي](/vertical-scaling/) حسب احتياج التطبيق للموارد الحاسوبية.
+
+## المشكلة التي حلتها
+الحاوية هي وحدة مستقلة قادرة على القيام بوظيفة معينة،
+وتحتاج بعد الوظائف إلى عدة حاويات تعمل بالتنسيق معاً للقيام بها.
```
Comment (arezk84) [inline content/ar/pod.md:15]: بعض وليس بعد

Comment (adowair) [reply content/ar/pod.md]: Thank you, done here: https://github.com/cncf/glossary/pull/2807/commits/53e9cd0ff2e7864a753453b8436cafc31111b05d

**Review by arezk84 — APPROVED** [review]

**Review by TarekMSayed — APPROVED** [review]

**Pull request merged**